### ISSUE TITLE
type 'Null' is not a subtype of type 'String' in type cast

### ISSUE THREAD

**Issue by mitchross** [fadhilx/json_to_model#34]
PS C:\Users\thell\Documents\Programming\flutter-apps\tvbc_flutter> flutter packages pub run json_to_model --source=".\.jsons\"
Unhandled exception:
type 'Null' is not a subtype of type 'String' in type cast
#0      defaultCommandCallback (package:json_to_model/core/command.dart:61:52)
#1      new DartDeclaration.fromCommand (package:json_to_model/core/dart_declaration.dart:313:35)
#2      new DartDeclaration.fromKeyValue (package:json_to_model/core/dart_declaration.dart:273:39)
#3      new JsonModel.fromMap.<anonymous closure> (package:json_to_model/core/json_model.dart:70:51)
#4      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:539:8)
#5      new JsonModel.fromMap (package:json_to_model/core/json_model.dart:69:13)
#6      JsonModelRunner._iterateOverJsonFiles.<anonymous closure> (package:json_to_model/index.dart:57:39)
#7      List.forEach (dart:core-patch/growable_array.dart:433:8)
#8      JsonModelRunner._iterateOverJsonFiles (package:json_to_model/index.dart:38:50)
#9      JsonModelRunner.run (package:json_to_model/index.dart:27:5)
#10     main (file:///C:/tools/flutter/.pub-cache/hosted/pub.dartlang.org/json_to_model-3.1.3/bin/json_to_model.dart:11:28)
<asynchronous suspension>

**Comment by bawantha** [comment]
any solution

**Comment by mark-nicepants** [comment]
Heya, can you provide your input json file?

**Comment by mitchross** [comment]
Hi @mark-apn 

Here is the JSON

```JSON
{
    "adult": false,
    "backdrop_path": "/zqkmTXzjkAgXmEWLRsY4UpTWCeo.jpg",
    "belongs_to_collection": {
        "id": 10,
        "name": "Star Wars Collection",
        "poster_path": "/tdQzRSk4PXX6hzjLcQWHafYtZTI.jpg",
        "backdrop_path": "/d8duYyyC9J5T825Hg7grmaabfxQ.jpg"
    },
    "budget": 11000000,
    "genres": [
        {
            "id": 12,
            "name": "Adventure"
        },
        {
            "id": 28,
            "name": "Action"
        },
        {
            "id": 878,
            "name": "Science Fiction"
        }
    ],
    "homepage": "http://www.starwars.com/films/star-wars-episode-iv-a-new-hope",
    "id": 11,
    "imdb_id": "tt0076759",
    "original_language": "en",
    "original_title": "Star Wars",
    "overview": "Princess Leia is captured and held hostage by the evil Imperial forces in their effort to take over the galactic Empire. Venturesome Luke Skywalker and dashing captain Han Solo team together with the loveable robot duo R2-D2 and C-3PO to rescue the beautiful princess and restore peace and justice in the Empire.",
    "popularity": 87.328,
    "poster_path": "/6FfCtAuVAW8XJjZ7eWeLibRLWTw.jpg",
    "production_companies": [
        {
            "id": 1,
            "logo_path": "/o86DbpburjxrqAzEDhXZcyE8pDb.png",
            "name": "Lucasfilm",
            "origin_country": "US"
        },
        {
            "id": 25,
            "logo_path": "/qZCc1lty5FzX30aOCVRBLzaVmcp.png",
            "name": "20th Century Fox",
            "origin_country": "US"
        }
    ],
    "production_countries": [
        {
            "iso_3166_1": "US",
            "name": "United States of America"
        }
    ],
    "release_date": "1977-05-25",
    "revenue": 775398007,
    "runtime": 121,
    "spoken_languages": [
        {
            "english_name": "English",
            "iso_639_1": "en",
            "name": "English"
        }
    ],
    "status": "Released",
    "tagline": "A long time ago in a galaxy far, far away...",
    "title": "Star Wars",
    "video": false,
    "vote_average": 8.2,
    "vote_count": 17259
}
```

**Comment by mark-nicepants** [comment]
Thanks for the example. I published an update 3.1.6 that will properly parse this json. It's already available on pub.dev

**Comment by mitchross** [comment]
@mark-apn  this 3.1.6 version fixed this issue, but now I have a new error https://github.com/fadhilx/json_to_model/issues/36
